server/service/src/preference/preferences/warn_when_missing_recent_stocktake.rs

```diff
@@ -0,0 +1,26 @@
+use crate::preference::{PrefKey, Preference, PreferenceType, PreferenceValueType};
+
+pub struct WarnWhenMissingRecentStocktake;
+
+#[derive(Default, Clone, Debug, PartialEq, serde::Serialize, serde::Deserialize)]
```

Comment:
My reckons is maybe here you need to give `serde` or `graphql` an attribute to serialise/rename fields to camelCase?